service/src/java/org/apache/hive/service/server/HiveServer2.java

```diff
@@ -442,6 +441,26 @@ public synchronized void init(HiveConf hiveConf) {
     ShutdownHookManager.addShutdownHook(() -> hiveServer2.stop());
   }
 
+  /**
+   * Checks if a String is blank. A blank string is one that is either
+   * {@code null}, empty, or all characters are {@link Character#isWhitespace(char)}.
+   *
+   * @param s the String to check, may be {@code null}
+   * @return {@code true} if the String is {@code null}, empty, or all characters are {@link Character#isWhitespace(char)}
+   */
+  private static boolean isBlank(final String s) {
+    if (s == null || s.isEmpty()) {
+      return true;
+    }
+    for (int i = 0; i < s.length(); i++) {
+      final char c = s.charAt(i);
+      if (!Character.isWhitespace(c)) {
+        return false;
+      }
+    }
+    return true;
+  }
+
```

Comment:
Can't we directly use ``StringUtils.isBlank()`` rather than implementing our own stuff? If we want to do so in that case we should put it in some Util class then....

Reply:
changed using new commit - to use commons-lang3 StringUtils.isBlank